sal_interfaces/MTVMS/MTVMS_Telemetry.xml

```diff
@@ -41,4 +41,72 @@
       <Count>50</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <Subsystem>MTVMS</Subsystem>
+    <EFDB_Topic>MTVMS_psd</EFDB_Topic>
+    <Description>VMS (1 sensor, 3 channels) accelerometer Power Spectral Densities (PSDs).</Description>
```

Comment:
I suggest removing VMS from the description. Perhaps something like "Acceleration power spectral density, measured by a 3-axis accelerometer". Perhaps this description can match the one for the ESS?

Reply:
Changed.